plugins/spring-remark-embed-video/index.js

```diff
@@ -16,9 +16,7 @@ module.exports = ({ markdownAST }, options = { width: 600, height: 300 }) => {
       const videoUrl = value.replace('<!--VIDEO:', '').replace('-->', '')
       if (isUrlValid(videoUrl)) {
         node.type = `html`
-        node.value = `<div><iframe src="${videoUrl}" width="${
-          options.width
-        }" height="${options.height}"></iframe></div>`
+        node.value = `<div><iframe src="${videoUrl}" width="${options.width}" height="${options.height}"></iframe></div>`
```

Comment:
Why do we need this?

Reply:
Have not noticed this was reformatted. Will revert it

Reply:
I still see the change on this file.